include/onnxruntime/core/common/common.h

```diff
@@ -242,4 +242,12 @@ inline std::wstring ToWideString(const std::wstring& s) { return s; }
 inline std::string ToWideString(const std::string& s) { return s; }
 #endif
 
+// Naming Convention for a Session Options Config Key
```

Comment:
How will the C API users discover these keys? This is a C++ file and won't be shipped as part of our packages. I suggest either (1) creating a separate file called session_options_config_keys.h and including it from the top level c_api.h or (2) adding the keys directly in c_api.h. I prefer 1 as it's a bit cleaner.